src/main/java/com/spacesongr/songr/controllers/Controller.java

```diff
@@ -0,0 +1,50 @@
+package com.spacesongr.songr;
+
+import com.spacesongr.songr.Repository.AlbumRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.ArrayList;
+
+@Controller
+public class Controller {
```

Comment:
Controller is a reserved term, so if you try to use it as a name for your controller class, you'll get errors.